RBSManager/Messages/TimeMessage.swift

```diff
@@ -9,8 +9,8 @@ import Foundation
 import ObjectMapper
 
 public class TimeMessage: RBSMessage {
-    public var sec: Float64 = 0
-    public var nsec: Float64 = 0
+    public var sec: Int32 = 0
+    public var nsec: UInt32 = 0
```

Comment:
Does `nsec` now need to be `nanosec` in ROS2? As per https://docs.ros2.org/galactic/api/builtin_interfaces/msg/Time.html